admin/services/data-access/setting.data.service.js

```diff
@@ -1,11 +1,18 @@
 'use strict'
 
 const Setting = require('../../models/setting')
+const sqlService = require('./sql.service')
 
 const settingDataService = {}
 
-settingDataService.findOne = async function (options) {
-  return Setting.findOne(options).lean().exec()
+settingDataService.findOne = async function () {
+  const sql = 'SELECT TOP 1 * FROM Settings'
+
+  const result = await sqlService.query(sql)
+  if (result && result.length === 1) {
```

Comment:
Why test for exactly one?  It could be `result.length > 0` I would understand it if we threw an error if there was more than one.